st2common/st2common/models/db/trigger.py

```diff
@@ -46,6 +46,7 @@ class TriggerTypeDB(stormbase.StormBaseDB,
     RESOURCE_TYPE = ResourceType.TRIGGER_TYPE
     UID_FIELDS = ['pack', 'name']
 
+    ref = me.StringField(required=False)
```

Comment:
Also see why this "ref" change is required. https://github.com/StackStorm/st2/issues/3454